cpu/cc3200/include/cc3200_prcm.h

```diff
@@ -0,0 +1,329 @@
+/*
+ * Copyright (C) 2019 Ludwig Maximilian Universität
+ *
+ * This file is subject to the terms and conditions of the GNU Lesser
+ * General Public License v2.1. See the file LICENSE in the top level
+ * directory for more details.
+ */
+
+/**
+ * @defgroup        cpu_cc3200_prcm CC3200 PRCM
+ * @ingroup         cpu_cc3200_regs
+ * @{
+ *
+ * @file
+ * @brief           Driver for the cc3200 PRCM controller
+ *
+ * Header file with register and macro declarations for the cc3200 PRCM module
+ *
+ * @author          Wladislaw Meixner <wladislaw.meixner@campus.lmu.de>
+ *
+ * @{
+ */
+
+#ifndef CC3200_PRCM_H
+#define CC3200_PRCM_H
+
+#include "cc3200.h"
+#include "vendor/hw_memmap.h"
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/**
+ * @brief convert x usecs to cpu cycles
+ *
+ */
+#define USEC_TO_CPU_CYCLES(x) (80 * x) / 3
```

Comment:
Shouldn't this take `CLOCK_CORECLOCK` into account?

Reply:
I could replace (80 * x)  with ((CLOCK_CORECLOCK / 1000000) * x) but importorting `periph_conf.h` causes some import/definition order troubles for me

Reply:
Huh, this should not happen.
What error do you get?